openlibrary/templates/book_providers/runeberg_read_button.html

```diff
@@ -0,0 +1,22 @@
+$def with(runeberg_id, analytics_attr)
+
+<div class="cta-button-group">
+  <a
+      href="https://runeberg.org/$runeberg_id/"
+      title="$_('Read eBook from Project Runeberg')"
+      class="cta-btn cta-btn--available cta-btn--read cta-btn--external cta-btn--runeberg"
+      target="_blank"
+      $:analytics_attr('Read')
+      aria-haspopup="true"
+      aria-controls="runeberg-toast"
+  >$_('Read')</a>
+</div>
+
+$if render_once('runeberg-toast'):
+  <div class="toast toast--book-provider" data-toast-trigger=".cta-btn--runeberg" id="runeberg-toast" style="display:none">
+    <div class="toast__body">
+      $:_('This book is available from <a href="https://runeberg.org/">Project Runeberg</a>. Project Runeberg is a trusted book provider of classic Nordic (Scandinavian) literature in electronic form.')
```

Comment:
Should the link/`href` attribute be dealt with differently than having it raw in the translatable string?

Reply:
I don't know the exact answer and I'm on my phone so I can't look at the code base too easily. But have you tried looking at other trusted book providers and the way they have this button?

Reply:
Looks like [`cita_press`](https://github.com/internetarchive/openlibrary/blob/0ed87f391f2f57cca16abaec0c58c9a521a3b01c/openlibrary/templates/book_providers/cita_press_read_button.html) and [`librivox`](https://github.com/internetarchive/openlibrary/blob/0ed87f391f2f57cca16abaec0c58c9a521a3b01c/openlibrary/templates/book_providers/librivox_read_button.html) both have the `href` directly in the translated string, so I guess this is fine.